lms/services/d2l_api/client.py

```diff
@@ -131,3 +153,14 @@ def get_api_user_id(user_id: str):
         so we don't take the second part of the string but the last.
         """
         return user_id.split("_")[-1]
+
+    def _get_course_modules(self, org_unit) -> List[dict]:
```

Comment:
I like to keep this method with the 1-1 relation to the API.

Keeping it private and not testing it independently in any case.